.github/workflows/release_flank_scripts.yml

```diff
@@ -1,4 +1,4 @@
-name: RELEASE
+name: Release Flank Scripts
```

Comment:
Do we still need this workflow?

Reply:
yes, it releases flank-scripts after each change

Reply:
Thanks!